packages/components/ng-ovh-mondial-relay/src/index.less

```diff
@@ -0,0 +1,103 @@
+@import '~bootstrap/less/variables.less';
+
```

Comment:
```css
@import (less) "~leaflet/dist/leaflet.css";
```

Already done from the host application so `leaflet` has been turned into a peerDependency.